src/main/java/frc/robot/Robot.java

```diff
@@ -31,12 +44,18 @@ public void disabledPeriodic() {
 
     @Override
     public void teleopInit() {
-
+        new InitializeArm(armSystem).start();
     }
 
     @Override
     public void teleopPeriodic() {
+        gripper.move(xboxController.getAxis(XboxAxis.LeftStickY).getAsDouble());
```

Comment:
Remove arm code from robot and use actions only. Though I noticed you already initialized a default action for the arm system, so can probably just remove this.